backend/database/seeding/helpers.go

```diff
@@ -228,12 +228,18 @@ func newFindingGen(first int64) func(opID int64, uuid string, category *int64, t
 	}
 }
 
-func newUserOpPermission(user models.User, op models.Operation, role policy.OperationRole) models.UserOperationPermission {
+func newUserOpPermission(user models.User, op models.Operation, role policy.OperationRole, favorite ...bool) models.UserOperationPermission {
+	isFavorite := false
+	if len(favorite) > 0 {
+		isFavorite = true
+	}
```

Comment:
```suggestion
	isFavorite := (len(favorite) > 0)
```

Reply:
or you could put it in the structure creation

Reply:
ah yeah that's much nicer - I think I'll put it in the struct